shapely/tests/geometry/test_feature.py

```diff
@@ -0,0 +1,14 @@
+import shapely.geometry
+
+
+def test_feature_from_geo_interface():
```

Comment:
There is a ``shapely/tests/legacy/test_shape.py`` file with existing tests for `shape(..)`, you can move it there I think